gpflux/layers/basis_functions/fourier_features/random/base.py

```diff
@@ -207,7 +245,10 @@ def build(self, input_shape: ShapeType) -> None:
         super(RandomFourierFeaturesCosine, self).build(input_shape)
 
     def _bias_build(self, n_components: int) -> None:
-        shape = (1, n_components)
+        if self.num_latent_gps is not None:
+            shape = (self.num_latent_gps, 1, n_components)
+        else:
+            shape = (1, n_components)  # type: ignore
```

Comment:
why is the ignore necessary?

Reply:
You get a weird typing error from mypy related to keras that I couldn't figure out how to solve